src/encoded/commands/generate_annotations.py

```diff
@@ -210,7 +208,7 @@ def human_annotations(human_file):
     """
     zipped_rows = get_rows_from_file(human_file, '\n')
     # Too many processes causes the http requests causes the remote to respond with error
-    pool = mp.Pool(processes=10)
+    pool = mp.Pool(processes=2)
```

Comment:
Why so much lower?

Reply:
More doesn't help, upstream API's throttle the requests anyways.